### DISCUSSION TITLE
可以增加一个AI图库整理功能么？

### DISCUSSION THREAD

**Discussion by axunrun** [mushan0x0/AI0x0.com#311] [Ideas]
目前在Stable diffusion针对图片AI识别后打TAG已经是很成熟的功能了，能否结合这个识别功能，设计出图库内的图片集打上英文TAG转中文TAG后的分类整理功能？设计行业从业者对这个工作苦不堪言，应该需求很大。我的图库3万多张参考图待整理。>_<

**Comment by mushan0x0** [comment]
可能不会开发，这个可能需要专门的软件来做体验应该比较好。。

**Reply by axunrun** [reply]
> 可能不会开发，这个可能需要专门的软件来做体验应该比较好。。

功能上已经有了，作为stablediffusion的附件功能，主要是针对用来训练模型的。可惜没有哪个开发对文件操作的功能，比如用tag名称建立文件夹然后移动文件这么简单一个事情。>_<

**Reply by axunrun** [reply]
> 可能不会开发，这个可能需要专门的软件来做体验应该比较好。。

另外问一下，目前文库功能只能用指定的一个具体文件而不能指向文件夹或则多层级文件夹？我想用GPT来查询我的工作目录，比如调出某客户项目最终报价之类的信息。

**Reply by mushan0x0** [reply]
后期解决性能和一些问题后会开放选择文件夹的
